src/bin/lfortran.cpp

```diff
@@ -1166,15 +1166,25 @@ int link_executable(const std::vector<std::string> &infiles,
                 return 10;
             }
         } else {
-            std::string CC = "clang";
-            char *env_CC = std::getenv("LFORTRAN_CC");
-            if (env_CC) CC = env_CC;
+            std::string CC;
             std::string base_path = "\"" + runtime_library_dir + "\"";
             std::string options;
             std::string runtime_lib = "lfortran_runtime";
-            if (compiler_options.target != "") {
-                options = " -target " + compiler_options.target;
-            }
+
+            if (link_with_gcc)
+              CC = "gcc";
+            else
+              CC = "clang";
```

Comment:
Style suggestion: Use {} and use 4 spaces:
```suggestion
            if (link_with_gcc) {
                CC = "gcc";
            } else {
                CC = "clang";
            }  
```